toolchain/lex/testdata/char_literals.carbon

```diff
@@ -105,8 +105,8 @@
 
 // This literal contains a raw tab character.
 // CHECK:STDERR: fail_invalid.carbon:[[@LINE+4]]:2: error: whitespace other than plain space must be expressed with an escape sequence in a string literal [InvalidHorizontalWhitespaceInString]
-// CHECK:STDERR: '{{\t}}'
-// CHECK:STDERR:  ^
+// CHECK:STDERR: '<09>'
```

Comment:
Have you considered prefixing these with `0x` in the output? I'm of two minds because it would be more verbose, but it seems like it would be clearer.